cfme/containers/node.py

```diff
@@ -27,16 +27,17 @@
 resource_locator = "//div[@id='records_div']/table//span[@title='{}']"
 
 
-class NodeView(BaseLoggedInPage):
-    title = Text('#explorer_title_text')
+class NodeView(ContainerObjectAllBaseView):
+    TITLE_TEXT = 'Nodes'
 
     monitor = Dropdown('Monitoring')
-    policy = Dropdown('Policy')
-    download = Dropdown('Download')
-
     nodes = Table(locator="//div[@id='list_grid']//table")
 
     @property
+    def table(self):
```

Comment:
@psav we need this for convenience, if you iterate over all container objects it'll cause spaghetti. e.g. [obj.table for obj in ContainersObjects], also, you can see that this is on latest already.